www/contribute/producing-an-ebook-step-by-step.php

```diff
@@ -191,7 +191,7 @@
 <span class="p">&lt;/</span><span class="nt">body</span><span class="p">&gt;</span>
 <span class="p">&lt;/</span><span class="nt">html</span><span class="p">&gt;</span></code></figure>
 				<p>If you look carefully, you’ll notice that the <code class="html"><span class="p">&lt;</span><span class="nt">html</span><span class="p">&gt;</span></code> element has the <code class="html"><span class="na">xml:lang</span><span class="o">=</span><span class="s">"en-US"</span></code> attribute, even though our source text uses British spelling! We have to change the <code class="html"><span class="na">xml:lang</span></code> attribute for the source files to match the actual language, which in this case is en-GB. Let’s do that now:</p><code class="terminal"><span><b>perl</b> -pi -e <i>"s|en-US|en-GB|g"</i> src/epub/text/chapter<i class="glob">*</i></span></code>
-				<p>Note that we <em>don’t</em> change the language for the metadata or front/back matter files, like <code class="path">content.opf</code>, <code class="path">titlepage.xhtml</code>, or <code class="path">colophon.xhtml</code>. Those must always be in American spelling, so they’ll always have the en-US language tag.</p>
+				<p>Note that we <em>don’t</em> change the language for the metadata or title page files, like <code class="path">content.opf</code>, <code class="path">colophon.xhtml</code>, <code class="path">imprint.xhtml</code>, <code class="path">toc.xhtml</code>, or <code class="path">titlepage.xhtml</code>. Those must always be in American spelling, so they’ll always have the en-US language tag.</p>
```

Comment:
Let's word this as `or boilerplate files` because the ToC/imprint/etc. are not really "title page files". Let's also cut  the explicit reference to content.opf because that's the only metadata file with lang anyway.

Reply:
Of course they're not "title page" files, only titlepage is, but I consider them metadata, or at least metadata-adjacent. Certainly more metadata than boilerplate. Uncopyright is boilerplate, these aren't that. Nevertheless, changes made.